core/src/main/java/org/elasticsearch/search/aggregations/AggregatorFactory.java

```diff
@@ -80,7 +80,10 @@ public boolean needsScores() {
 
         @Override
         public Aggregator subAggregator(String name) {
-            throw new UnsupportedOperationException();
+            if(aggregators.get(0) != null) {
+                return aggregators.get(0).subAggregator(name);
```

Comment:
I'm not sure this will work for all cases since this is always returning the first aggregator in the list (the aggregator relating to the parent bucket which has ordinal `0`). I think by doing this it will return the wrong aggregator (and therefore give unexpected results) when the aggregator object for other parent bucket ordinals are required.